third_party/txt/src/txt/font_collection.cc

```diff
@@ -164,6 +160,10 @@ FontCollection::GetMinikinFontCollectionForFamilies(
       }
     }
   }
+  // Default font family also not found. We fail to get a FontCollection.
```

Comment:
Fallback families are intended to provide additional characters that aren't handled by the primary font (e.g. emoji or CJK).

The original logic would return failure if the primary font can not be resolved.  I think we should continue to do that.